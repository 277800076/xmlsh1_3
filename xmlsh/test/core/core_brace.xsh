# core_brace.xsh

# Non-outputing test
{ false; }
{ false ; } && { echo invalid return ; exit 1 ; }
{ false ; true ; }


# multi line

{ 
	true ; 
	false ;
}

# multi line with if

true && {
	true ; 
	false ;
}

# simple output

echo {  not reserved  middle or end of command }


{ echo true ;  }
{ echo line1 ; echo line2 ; }

# xml piping
{ echo '<foo/>' ; } | xcat

C=echo
${C}
echo '<foo2/>' | { xcat | xcat;}




# { env does change
A=1
{ A=2  }

if [ ! $A = 2 ] ; then 
	echo  Brace environment wrong A=$A
fi



exit 0

