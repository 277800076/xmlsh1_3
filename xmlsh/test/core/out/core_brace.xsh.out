{ not reserved middle or end of command }
true
line1
line2
<foo/>
<foo2/>
test
A is 2
